src/elements/Box.tsx

```diff
@@ -56,23 +55,6 @@ export const BorderBox = styledWrapper(Flex).attrs<BorderBoxProps>({})`
   ${width};
 `
 
-// export const StackableBorderBox = styled(BorderBox)`
```

Comment:
Given `StackableBorderBox` isn't built in a way that can work on RN atm, I think it's fine to just keep that in Reaction until we can figure out a good way to make it universal.